.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+/web/ui @juliusv
+/storage/remote @csmarchbanks @cstyan @bwplotka
```

Comment:
```suggestion
/storage/remote @csmarchbanks @cstyan @bwplotka @tomwilkie 
```

Reply:
Do you also want to be in MAINTAINERS.md?

Reply:
Sure, happy to.